src/services/userService.ts

```diff
@@ -0,0 +1,57 @@
+import { sign } from "jsonwebtoken";
+import { compare, hash } from "bcryptjs";
+import IUser from "interfaces/userInterface";
+import { User } from "models/userModel";
+import { appConfig } from "config/app-config";
+
+export const createUser = async (obj: IUser) => {
+  obj.password = await hash(obj.password, 10);
+  const user = await User.create({ ...obj });
+  return user;
+};
+
+export const addUser = async (obj: IUser) => {
+  const user: IUser[] = await User.find({ name: obj.name });
+  if (!user) {
+    return new Error("Bad Request");
+  }
+  const passcheck = await compare(obj.password, user[0].password);
+  if (passcheck) {
+    const Token = sign({ name: user[0].name }, appConfig.secretkey as string, {
+      expiresIn: "1h",
+    });
+    return Token;
+  } else {
+    return new Error("Wrong Pass");
+  }
+};
+
+export const findAllUsers = async () => {
+  const users = await User.find({});
+  return users;
+};
```

Comment:
direct return no need to create variable and then return you can return directly.

Reply:
I mentioned this comment before also.